src/components/ConnectionInvitation.component.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC } from 'react';
+import Avatar from './Avatar.component';
+import Button from './Button.component';
+
+interface Props {
+  userId: string;
+  avatar: string;
+  username: string;
+  notifiedAt: string;
+}
+
+const ConnectionInvitation: FC<Props> = ({
+  userId,
+  avatar,
+  username,
+  notifiedAt,
+}) => {
+  const handleAccept = () => {
+    console.log(`Invitación aceptada para el usuario con ID: ${userId}`);
+  };
+
+  const handleReject = () => {
+    console.log(`Invitación rechazada para el usuario con ID: ${userId}`);
+  };
+
+  const redirectToProfile = () => {
+    window.location.href = `/${userId}/profile`;
+  };
+
+  return (
+    <div className="mx-auto d-flex align-items-center border rounded p-3 bg-white text-dark" style={{ maxWidth: '500px' }}>
```

Comment:
No debes limitar el width, en ningún momento lo he dicho 

```suggestion
    <div className="mx-auto d-flex align-items-center border rounded p-3 bg-white text-dark">
```